### PR TITLE
Js faq start

### PR DESCRIPTION
**Pull request by jsingle**
@paetling documentation PR :yey:

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### I'm having trouble running docker commands on my mac - what gives??
+If you're seeing something like:
+
+```
+$ docker ps
+FATA[0000] Get http:///var/run/docker.sock/v1.18/containers/json: dial unix /var/run/docker.sock: no such file or directory. Are you trying to connect to a TLS-enabled daemon without TLS?
+```
+
+This isn't Dusty's fault. You probably just need to run `$(boot2docker shellinit)`
```
Comment (thieman) [inline docs/faq.md]: No need to get all defensive there Gary

**Comment by paetling** [comment]
seems fine if sparse

File: docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+Daemon will make use of your unpriveleged user's SSH_AUTH_SOCK. This means that the Daemon's
+behavior using git should be the same as your user's, in terms of permissions.
+
+If you want Dusty to be
+able to clone your private Github repos, you need to configure your standard user (whoever
+your `mac_username` is set to), to be able to clone repositories without any prompt for
+confirmation or password.
+
+See [this Github help article](https://help.github.com/articles/generating-ssh-keys/)
+for some info about setting up Github ssh keys.
```
Comment (thieman) [inline docs/faq.md]: GitHub

**Comment by jsingle** [comment]
What does "sparse" mean?

**Pull request merged**